gen/calc.py

```diff
@@ -605,7 +605,7 @@ def validate_exhibitor_storage_master_discovery(master_discovery, exhibitor_stor
                 }
             }
         }),
-        'enable_gpu_isolation': 'false',
+        'enable_gpu_isolation': 'true',
```

Comment:
That is from my PR https://github.com/dcos/dcos/pull/1227. It looks like jeff maybe rebased on my branch. And if so, he'll need to do it again since I just rebased on master this morning. Also, the description should say that you are rebased on my PR in order to remove confusion like this and track the depenedency. Unfortunately, github doesn't let you base your PR on another PR (only on other branches), otherwise we wouldn't need to track the dependencies so manually.